tests/test_doc_examples.py

```diff
@@ -26,9 +28,9 @@
     skipif = lambda func, reason="": func  # noqa E731
 
 
-def get_doc_examples():
+def get_doc_examples(resources):
     policies = []
-    for resource_name, v in resources().items():
+    for resource_name, v in resources.items():
```

Comment:
we should do a class seen list here to minimize dupes across common filters/actions.

Reply:
that actually speeds up the processing here by at least a dozen seconds just avoiding the yaml.load